js/modules/k6/experimental/tracing/module.go

```diff
@@ -38,6 +40,11 @@ func New() *RootModule {
 // NewModuleInstance implements the modules.Module interface and returns
 // a new instance for each VU.
 func (*RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
+	// Seed the random number generator with the current time.
+	// This ensures that any call to rand.Intn() will return
+	// less-deterministic results.
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
`rand` is a global so this likely should be a field in the `ModuleInstance` see my other commetn

Reply:
Indeed 👍🏻